community/bkeymaps/post-install

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh -e
+
+echo "\
+NOTE: Keymaps are copied to /usr/share/bkeymaps.
+
+NOTE: You should pipe the keymaps to loadkmap in 
+NOTE: order to load them.
+
+NOTE: example usage:
+NOTE:      loadkmap < /usr/share/bkeymaps/us/us.bmap"
```

Comment:
Look at how this is done in other `post-install` files or swap to using `printf`. :+1:

It's also worth updating the Wiki page to make note of the existence of this package.

Reply:
I was already writing a wiki page for my own site, I'll add it to yours as well!